Fixes for colony gamemode.

Flaksim opened the pull request:
-maptweak: "Remodeled frigate deck 4 to facilitate readding ODST's."
  -rscdel: "Removed ODST spawns on prowler."
  -rscadd: "Added ODST spawns to frigate."
  -rscadd: "Added all First Contact jobs and spawns to colony gamemode."
  -rscadd: "Changed ONI Bridge Officer to ODST Officer."

Flaksim committed:
Fixes loading list for ONI Guards.

*Fixes spawn list so ONI guards no longer spawn with ONI Guard Squad Leader loadout.

Flaksim committed:
Merge pull request #1 from HaloSpaceStation/alpha

update fork

Flaksim committed:
Merge pull request #2 from HaloSpaceStation/alpha

update fork

Flaksim committed:
Fixes to colony gamemode

  -maptweak: "Remodeled frigate deck 4 to facilitate readding ODST's."
  -rscdel: "Removed ODST spawns on prowler."
  -rscadd: "Added ODST spawns to frigate."
  -rscadd: "Added all First Contact jobs and spawns to colony gamemode."
  -rscadd: "Changed ONI Bridge Officer to ODST Officer."

bloxgate commented on html/changelogs/Flaksim Update 5 june.yml:
```diff
@@ -0,0 +1,40 @@
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  -maptweak: "Remodeled frigate deck 4 to facilitate readding ODST's."
```
There needs to be a space between the - and the letter

Flaksim replied:
Added.

Flaksim committed:
Fixes travis errors

*Added the corvette spawns to the frigate def includes
*Fixed changelog layout.

Flaksim committed:
Fixes impure area.

Area added to frigate code and corrected on map.
